Digipost.Signature.Api.Client.Direct.Tests/DataTransferObjects/DataTransferObjectConverterTests.cs

```diff
@@ -177,22 +180,20 @@ public void Converts_direct_job_status_with_multiple_signers_successfully()
             public void Converts_document_successfully()
             {
                 //Arrange
-                const string subject = "Subject";
+                const string title = "JobTitle";
                 const string message = "Message";
```

Comment:
Github actions hinter om 
`The variable 'message' is assigned but its value is never used`